tools/local-network/README.md

```diff
@@ -36,21 +36,34 @@ This is a simple script to bootstrap a test ledger and keys to play with. The sc
 
 This script starts a local mobilecoin consensus network by launching a separate process for each consensus validator and configuring them to communicate via a default set of ports. It takes the following parameters:
 
-- (required) `--network-type` - describes the network topology, one of `dense5`, `a-b-c`, `ring5` or `ring5b`
+- (required) `--network-type` - describes the network topology, one of `dense5`, `dense3`, `a-b-c`, `ring5` or `ring5b`
 - (optional) `--skip-build` - does not rebuild consensus node binaries
 - (optional) `--block-version` - specifies local network block version (defaults to highest available if not specified)
 
 It relies on environment variables for configuration:
 
 - (required) `LEDGER_BASE` - Points at the ledger directory to initialize the nodes with (e.g. `./target/sample_data/ledger`).
-- (optional) `IAS_API_KEY` - IAS Api key.
-- (optional) `IAS_SPID` - IAS Service Provider ID.
+- (optional) `IAS_API_KEY` - IAS Api key. (Only needed for IAS prod builds)
```

Comment:
Again maybe late to the party here, Is that correct? IAS Dev still requires spid and key.  It wouldn't be required for a `SGX_MODE=SW` build.

Reply:
I see -- thank you
I can fix this code comment, I think what happens is the script defaults it to all 0s if it is omitted and that works in the normal configuration of the script